install/kubernetes/cilium/values.yaml

```diff
@@ -1350,10 +1351,11 @@ sockops:
   # enabled enables installation of socket options acceleration.
   enabled: false
 
-# TODO: Add documentation, default value
-# svcSourceRangeCheck:
+# -- Enable check of service source ranges (currently, only for LoadBalancer) (default true).
```

Comment:
I don't think it's necessary to provide the default value between parenthesis, here and elsewhere. In the current file, the default is right on the following line; in the generated documentation, it's picked up and displayed already, no need to have it a second time.